_posts/en/newsletters/2019-10-23-newsletter.md

```diff
@@ -0,0 +1,127 @@
+    skilled Bitcoin users to dedicate time to reviewing the taproot set
+    of proposals.
+
+    Anyone wanting to participate should [RSVP][trc rsvp] soon so that the
+    organizers can estimate the total number of participants and start
+    forming study groups.  To register or learn more, please see
+    the [Taproot Review Club][trc] repository.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin wallets
+and services.*
+
+- **Electrum Lightning support:** In a [series of commits this
+  month][electrum commits], Electrum has merged support for Lightning
+  Network into master. A presentation deck titled [Lightning Implementation in
+  Electrum][electrum lightning presentation] by Thomas Voegtlin provides some
+  background information and screenshots.
+
+- **Blockstream Green Tor support:** Version 3.2.4 of the Blockstream Green
+  Wallet [adds built-in Tor support][blockstream green tor article] for both iOS and Android.
+  While Tor was supported in previous Android versions, it required a separate
+  application whereas both Android and iOS versions are now bundled with Tor
+  support.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3150][] adds new `signmessage` and `checkmessage` RPCs.
+  The first RPC will sign a message that can be verified by someone with
+  your LN node's public key.  The second RPC will verify a signed message
+  from another node either using a pubkey provided by the user or by
+  confirming the message is signed by a pubkey belonging to any known LN
+  node (e.g. a node in the set returned by the `listnodes` RPC).
+
+- [LND #3595][] raises the default maximum CLTV expiry from 1,008 blocks
+  (about 1 week) to 2,016 blocks (about two weeks).  This is the maximum
+  amount of time a new payment can be stuck as pending before it can be reclaimed
+  by its spender.  LND
+  recently tried to keep this at 1,008 blocks by decreasing the CLTV
+  delta---the minimum number of blocks each routing node along the payment path
+  has to claim its particular payment---from 144 blocks to 40 blocks
+  (see [Newsletter #40][lnd cltv delta]) but older LND nodes and some
+  other implementations have continued to use 144 as their default.  If
+  each hop requires a delta of 144, the new maximum expiry of 2,016 makes the maximum-length
+  routing path about 14 hops.
+
+- [LND #3597][] reverts the migration policy described in [Newsletter
+  #64][lnd3485] where LND could only be upgraded from a maximum of one major
+  release back.  The PR for the reversion notes, "the prior stricter
+  policy created a large burden on applications that package lnd as
```

Comment:
Ah yes, there was some strong blowback from application developers on that prior upgrade change.